serenity-automation-mentoring/src/test/java/serenity/EnvironmentPropertyLoader.java

```diff
@@ -0,0 +1,56 @@
+package serenity;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+
+
+public class EnvironmentPropertyLoader {
+    private static Logger log = LoggerFactory.getLogger(EnvironmentPropertyLoader.class);
+    private static EnvironmentPropertyLoader instance = null;
+    public static final String DEFAULT_CONFIG = "environments/env1.properties";
+    public static final String ENVIRONMENT_CONFIG_SYSTEM_PROPERTY = "environment.config";
+    private final Properties properties = new Properties();
+
+    private EnvironmentPropertyLoader() {
+        try {
+            if (System.getProperty(ENVIRONMENT_CONFIG_SYSTEM_PROPERTY) == null) {
+                this.loadProperties(DEFAULT_CONFIG);
+            } else {
+                this.loadProperties("environments/" + System.getProperty(ENVIRONMENT_CONFIG_SYSTEM_PROPERTY));
+            }
+
+        } catch (IOException var2) {
```

Comment:
name var2 is taken from de-compiled code. Please use meaningful namings. For exceptions it is `e` or `ex`

Reply:
Done